dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js

```diff
@@ -1,98 +1,183 @@
-/* eslint-disable mocha/no-setup-in-describe */
 import { cmpIframe } from '../../lib/cmpIframe';
 import { privacySettingsIframe } from '../../lib/privacySettingsIframe';
 import { storage } from '@guardian/libs';
 
-const mainMediaVideo =
-	'https://www.theguardian.com/uk-news/2020/dec/04/edinburgh-hit-by-thundersnow-as-sonic-boom-wakes-residents';
-const mainMediaPlayEvent = {
-	id: 'gu-video-youtube-2b33a7b7-e639-4232-9ecd-0fb920fa8147',
-	eventType: 'video:content:play',
-};
-
-const embedMediaVideo =
-	'https://www.theguardian.com/us-news/2020/oct/29/lordstown-ohio-trump-gm-plant-election';
-const embedPlayEvent = {
-	id: 'gu-video-youtube-35f9a38f-c0c5-4bbd-9395-e338c9e97d30',
-	eventType: 'video:content:play',
+const interceptPlayEvent = (id) => {
+	return cy.intercept(
+		{
+			url: 'http://ophan.theguardian.com/img/2?**',
+			query: {
+				video: /(.*)eventType\":\"video:content:play(.*)/,
+			},
+		},
+		function (req) {
+			const url = new URL(req.url);
+			const videoValue = url.searchParams.get('video');
+			expect(JSON.parse(videoValue)).to.deep.equal({
+				id,
+				eventType: 'video:content:play',
+			});
+		},
+	);
 };
-
-describe('Video', function () {
+describe('YouTube Atom', function () {
 	beforeEach(function () {
 		storage.local.set('gu.geo.override', 'GB');
 	});
 
-	it('Main media youtube video', function () {
-		cy.visit(`/Article?url=${mainMediaVideo}`);
+	it('plays main media videos', function () {
+		cy.visit(
+			'/Article?url=https://www.theguardian.com/uk-news/2020/dec/04/edinburgh-hit-by-thundersnow-as-sonic-boom-wakes-residents',
+		);
 		cmpIframe().contains("It's your choice");
 		cmpIframe().find("[title='Manage my cookies']").click();
 		privacySettingsIframe().contains('Privacy settings');
-		privacySettingsIframe().find("[title='Accept all']").click();
+		privacySettingsIframe()
+			.find("[title='Accept all']", { timeout: 12000 })
```

Comment:
Nifty little formatting syntax which makes large numbers easier to parse:

```suggestion
			.find("[title='Accept all']", { timeout: 12_000 })
```

Reply:
I implemented this as it's a nice feature but I had to revert as it was throwing lint errors 😞